ydb/core/scheme/scheme_tablecell.h

```diff
@@ -415,6 +415,8 @@ static_assert(std::is_nothrow_default_constructible_v<TOwnedCellVec>, "Expected
 // When loading from a buffer the cells will point to the buffer contents
 class TSerializedCellVec {
 public:
+    explicit TSerializedCellVec(const TConstArrayRef<TCell>& cells);
```

Comment:
I'm on a fence about this one. I think we avoided this constructor in the past, because it's very expensive and we didn't want to use it unintentionally, but it's probably not a problem in practice. But in any case `TConstArrayRef` should always be passed by value, never by reference.

Reply:
Updated.